parlai/core/torch_agent.py

```diff
@@ -46,6 +47,9 @@
 except ImportError:
```

Comment:
i think if we have this `check_torch_version` perhaps we can just move this try/except to there totally, and just have `import torch` here

Reply:
Yes, I should probably remove the try catch here and add the try catch within the ```check_torch_version```

Reply:
So the isse mentioned that we get rid of all try-catch around the imports since torch is now a requirement. So I can remove the try catch here and replace it with a regular import, but I am not sure if I should have the try-catch in our function, since I thought the purpose of this PR was to remove ll try-catches. Do let me know if I am wrong and you would like it otherwise!

Reply:
I think we want to keep *1* try catch so that we can fail on our own terms (we still want to fail) - i.e. fail with appropriate instructions, rather than "Module Not Found"